src/backend/_scss/application.scss

```diff
@@ -525,35 +527,56 @@ a {
   width: 355px;
   z-index: 1000;
 
+  .pop>hr {
+    margin: 10px 0;
+    min-width: calc(100% - 20px);
+  }
+
+  .pop>h4 {
+    margin-left: -20px;
+  }
+
+  .pop>.pop-over.row {
+    padding-bottom: 0;
+    padding-left: 0;
+    padding-right: 20px;
+    padding-top: 10px;
+  }
+
   .pop {
     background-color: $white;
+    margin-left: 20px;
+
+    a {
+      margin-left: -5px;
+    }
+
+    .pop-over>.col-xs-12,
+    .pop-over>.col-md-12 {
+      padding: 0;
+    }
 
     .pop-over {
       color: $gray-dark;
       font-weight: 400;
       line-height: 20px;
-      margin: 10px;
-      padding: 2%;
+      margin: 0;
+      padding: 0;
       word-wrap: break-word;
     }
+  }
 
-    .pop-footer {
-      background: $white;
-      border-top-left-radius: 5px;
-      border-top-right-radius: 5px;
-      color: $gray-dark;
-      line-height: 0.80em;
-      padding: 20px;
-    }
-
-    .pop-header {
-      background: $span-background;
-      border-bottom: 1px solid $gray-perfect;
-      border-top-left-radius: 5px;
-      border-top-right-radius: 5px;
-      height: 35px;
-      padding: 5px 5px 0 20px;
-    }
+  .pop-header {
+    background: $span-background;
+    border-bottom: 1px solid $gray-perfect;
+    border-top-left-radius: 5px;
+    border-top-right-radius: 5px;
+    height: 35px;
+    margin-left: -20px;
+    padding-bottom: 0;
```

Comment:
@aayusharora The scss linter pointed out that the shorthand can't have 4 arguments hence I had to put them in separately. Is there a different way to do it?